staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -188,6 +189,7 @@ type PodFailurePolicyOnExitCodesRequirement struct {
 	//   by the 'containerName' field) is not in the set of specified values.
 	// Additional values are considered to be added in the future. Clients should
 	// react to an unknown operator by assuming the requirement is not satisfied.
+	// +required
```

Comment:
Error to be empty https://github.com/kubernetes/kubernetes/blob/e34f01402f2be14e68b03a58c054f1a86bff55b9/pkg/apis/batch/validation/validation.go#L377-L378
Agreed required